src/ViewNamespaces.php

```diff
@@ -55,7 +55,8 @@ private static function getForDomain(string $domain)
      */
     private static function getFromConfigFor(string $domain, $customConfigKey = null)
     {
-        return config($customConfigKey ?? 'backpack.crud.view_namespaces.'.$domain) ?? [];
+        return config($customConfigKey ?? 'backpack.crud.view_namespaces.'.$domain) ??
+            [config('backpack.base.view_namespace').$domain];
```

Comment:
This is QUESTIONABLE. Should re-evaluate.

- before: when a widget/field/etc didn't exist, we defaulted to `[]`
- after: when a widget/field/etc doesn't exist, we look inside the theme directory, for a folder with the domain name (eg. `widgets`, `fields`, `columns` etc);

Reply:
Note to self: this one line took 1.5 hours to write 😅 Take it seriously.